api/DOMTokenList.json

```diff
@@ -76,10 +76,10 @@
               "version_added": "45"
             },
             "safari": {
-              "version_added": true
+              "version_added": "10.1"
```

Comment:
I think there might be a little discrepancy here.  WebKit versions are the same for Safari Desktop and iOS in Safari 10.  After that, Safari Desktop 10.1 is the same as iOS 10.3.  Can we double-check to make sure these are the correct versions?

Reply:
Discrepancy is here, it's true 😄And Apple's sources prove this fact, see [my comment](https://github.com/mdn/browser-compat-data/pull/4628#issuecomment-522816445).